vendor/github.com/moby/buildkit/vendor.conf

```diff
@@ -4,9 +4,9 @@ github.com/pkg/errors v0.8.0
 github.com/stretchr/testify v1.1.4
 github.com/davecgh/go-spew v1.1.0
 github.com/pmezard/go-difflib v1.0.0
-golang.org/x/sys 314a259e304ff91bd6985da2a7149bbf91237993
+golang.org/x/sys 1b2967e3c290b7c545b3db0deeda16e9be4f98a2
 
-github.com/containerd/containerd b41633746ed4833f52c3c071e8edcfa2713e5677
+github.com/containerd/containerd a88b6319614de846458750ff882723479ca7b1a1
```

Comment:
looks like we're behind here: https://github.com/containerd/containerd/compare/b41633746ed4833f52c3c071e8edcfa2713e5677...a88b6319614de846458750ff882723479ca7b1a1

Reply:
i've add another commit to update containerd vndr

Reply:
@dmcgowan i see the `containerd.WithExit` was removed in containerd https://github.com/containerd/containerd/commit/da1b5470cd26e9e2b4b1cd0100c876ce8db87471 as a result of the latest Runtime v2 changes of containerd. 

[janky](https://jenkins.dockerproject.org/job/Docker-PRs/50279/console) no likey:
```
libcontainerd/client_daemon.go:566:23:warning: WithExit not declared by package containerd (gosimple)
cmd/dockerd/daemon_unix.go:13:2:warning: could not import github.com/containerd/containerd/runtime/linux (cannot find package "github.com/containerd/containerd/runtime/linux" in any of: (interfacer)
```

Reply:
i'll remove the update containerd vndr commit 3763384 for now so we can proceed with this PR if it is not needed to get buildkit updated.

Reply:
`WithExit` -> `WithExit("")` should do the trick here. @crosbymichael do you have a better suggestion for a value other than `""`, looks like that will default to the runc types which were previously used.